install/example/freebsd_freenas.md

```diff
@@ -1,202 +1,409 @@
+```
+# cd /iocage/jails
+/iocage/jails # cp build-ffmpeg/root/usr/ports/multimedia/ffmpeg/work/pkg/ffmpeg-4.2.2_5,1.txz airsonic-jail/root/root/
+/iocage/jails # build-ffmpeg/root/usr/ports/audio/lame/work/pkg/lame-3.100_2.txz airsonic-jail/root/root/
+```
 
-Symlink ffmpeg to where Airsonic expects the transcoder to be.
+Once the packages are successfully copied to your Airsonic jail, you should see it in your 
+root user home directory. Open a shell on your Airsonic Jail and install the packages. 
+ffmpeg requires lame, so install lame first.
+```
+# pkg add /root/lame-3.100_2.txz
+Installing lame-3.100_2...
+Extracting lame-3.100_2: 100%
 
+# pkg add /root/ffmpeg-4.2.2_5,1.txz
+Installing ffmpeg-4.2.2_5,1...
+Extracting ffmpeg-4.2.2_5,1: 100%
+```
+
+Because we don't want the pre-packaged ffmpeg to overwrite our custom ffmpeg package, we 
+will lock the packages:
+```
+# pkg lock ffmpeg
+ffmpeg-4.2.2_5,1: lock this package? [y/N]: y
+Locking ffmpeg-4.2.2_5,1
+# pkg lock lame
+lame-3.100_2: lock this package? [y/N]: y
+Locking lame-3.100_2
+```
+
+Make sure you've got proper permissions on the ffmpeg executable:
+```
+# chown www:www /usr/local/bin/ffmpeg
+```
+
+Symlink ffmpeg to where Airsonic expects the transcoder to be.
 ```
 ln -s /usr/local/bin/ffmpeg /var/airsonic/transcode/ffmpeg
 ```
 
 Finally restart tomcat
+```
+# service tomcat85 restart
+```
+
+Congratulations you have ffmpeg with mp3 support installed ready for Airsonic to use!
+
+#### Enable Tomcat web administration gui
+
+Edit Tomcat's user configuration file with your favourite text editor. We installed nano in step 1.
 
 ```
-service tomcat8 restart
+rm /usr/local/apache-tomcat-8.0/conf/tomcat-users.xml
+nano /usr/local/apache-tomcat-8.0/conf/tomcat-users.xml
 ```
 
-Congratulations you have ffmpeg with mp3 support installed ready for Airsonic to use.
+Copy/paste the following:
+
+```
+<tomcat-users xmlns="http://tomcat.apache.org/xml"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://tomcat.apache.org/xml tomcat-users.xsd"
+version="1.0">
+
+<role rolename="manager-gui"/>
+<role rolename="manager-script"/>
+<role rolename="manager-jmx"/>
+<role rolename="manager-status"/>
+<role rolename="admin-gui"/>
+<role rolename="admin-script"/>
+<user username="admin" password="admin" roles="manager-gui,manager-script,manager-jmx,manager-status,admin-gui,admin-script"/>
+
+</tomcat-users>
+```
+
+> **NOTE**: If you wish to use a different username and password please append the second last line to contain your preferred username and password.
+> ```
+> <user username="yourusername" password="yourpassword" roles="manager-gui,manager-script,manager-jmx,manager-status,admin-gui,admin-script"/>
+> ```
+
+By default Tomcat only allows uploading of files up to 50mb. Our WAR file is slightly larger so we have to modify the according setting. Open the configuration file for the web-interface:
```

Comment:
Remove `of`

Reply:
done lol, it was a holdover from the previous guide.